cmd/eh-frame/main.go

```diff
@@ -27,6 +27,7 @@ import (
 type flags struct {
 	Executable string `kong:"help='The executable to print the .eh_unwind tables for.'"`
 	Compact    bool   `kong:"help='Whether to use the compact format.'"`
+	RelativePc uint64 `kong:"help=''"`
```

Comment:
⛏ nitpick
```suggestion
	RelativePC uint64 `kong:"help=''"`
```

Reply:
No strong opinions here, but I thought [we discussed](https://github.com/parca-dev/parca-agent/pull/1004#discussion_r1017847289) that we preferred camelcase even for acronyms. Which one would you prefer that we use?

Reply:
Oops, sorry 😅 This keeps triggering my alerts because it's not the idiomatic way to name abbreviations in Go.
Thanks for baring with my nitpicks 😁

Reply:
No worries! If we decide to change this we can perhaps mass-change all the offences in another PR?

Reply:
Sounds good to me. 

But again, I don't have any problems if you don't get frustrated with my autonomous PR comments (some part of my brain acts as a linter 😁). Feel free to save the above comment as a template 😜